enterprise_access/apps/subsidy_access_policy/constants.py

```diff
@@ -55,3 +55,26 @@ class PolicyTypes:
 ]
 
 POLICY_TYPE_FIELD_MAPPER = dict(zip(POLICY_TYPES_WITH_CREDIT_LIMIT, POLICY_TYPE_CREDIT_LIMIT_FIELDS))
+
+class TransactionStateChoices:
+    """
+    Lifecycle states for a ledger transaction (i.e., redemption).
+
+    CREATED
+        Indicates that the transaction has only just been created, and is the default state.
+
+    PENDING
+        Indicates that an attempt is being made to redeem the content in the target LMS.
+
+    COMMITTED
+        Indicates that the content has been redeemed, and a reference to the redemption result (often an enrollment ID)
+        is stored in the reference_id field of the transaction.
+
+    FAILED
+        Indidcates that the attempt to redeem the content in the target LMS encountered an error.
+    """
+
+    CREATED = 'created'
+    PENDING = 'pending'
+    COMMITTED = 'committed'
```

Comment:
Only "committed" is used in this code base right now, but if we're bringing in one of the states, might as well bring them all in so they're available.